test/docfx.Test/lib/JsonSchemaTest.cs

```diff
@@ -141,16 +137,26 @@ public void TestJsonSchemaConfirmance(string description, string schemaText, str
         // string length validation
         [InlineData("{'type': 'string', 'minLength': 1, 'maxLength': 5}", "'a'", "")]
         [InlineData("{'type': 'string', 'maxLength': 1}", "'1963-06-19T08:30:06Z'",
-            "['warning','string-length-invalid','String length should be <= 1','file',1,22]")]
+            "['warning','string-length-invalid','String '' length should be <= 1','file',1,22]")]
```

Comment:
is this a property name or just empty string?

Reply:
The property name, but the message indeed is ambiguous